Library/Homebrew/dev-cmd/pr-publish.rb

```diff
@@ -16,6 +16,10 @@ def pr_publish_args
       EOS
       flag   "--tap=",
              description: "Target tap repository (default: `homebrew/core`)."
+      flag   "--workflow=",
+             description: "Target workflow filename (default: `publish-commit-bottles.yml`)."
+      flag   "--ref=",
+             description: "Target ref name (default: `master`)."
```

Comment:
When would you use a different branch here?

Reply:
Good question. For now I don't see any usages. Can just delete this option altogether and default to `master` always.

Reply:
Yeh, just delete for now I reckon then 👍🏻